### ISSUE TITLE
Admonition blockquotes should be converted to divs

### ISSUE THREAD

**Issue by voughtdq** [elixir-lang/ex_doc#1939]
For accessibility reasons, admonition blockquotes should be converted to divs.

**Comment by whatyouhide** [comment]
Closing in favor of the PR 🙃
